lib/cext/preprocess.rb

```diff
@@ -8,7 +8,7 @@
 
 ARGF.each do |line|
   if line.include?('rb_scan_args')
-    puts line.gsub(/rb_scan_args\((\w+), (\w+), \"(.*)\", /) {
+    puts line.gsub(/\srb_scan_args\((\w+), (\w+), \"(.*)\", /) {
```

Comment:
But `_` for example is a boundary character, so it would match `util_spec_rb_scan_args`.